docker-compose.yaml

```diff
@@ -0,0 +1,13 @@
+version: "3"
+
+services: 
+  vrising: 
+    image: public.ecr.aws/r0w6f3t3/vrising:latest
+    args:
+        STEAM_EPOC:
```

Comment:
I think you can remove this again. When I initially created the Dockerfile, I used another Dockerfile as the base and this was just a remnant.